types/oauth/index.d.ts

```diff
@@ -12,10 +12,10 @@ export type oauth1tokenCallback = (
 ) => any;
 
 export type oauth2tokenCallback = (
-    err: { statusCode: number; data?: any },
-    access_token: string,
-    refresh_token: string,
-    result: any,
+    err: { statusCode: number; data?: any } | null | unknown,
```

Comment:
`{ statusCode: number; data?: any } | null | unknown` evaluates to `unknown` so you can just change that to `unknown`